File: 24_C#Fundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_01_Lecture/_11_MultiplicationTable2/_11_MultiplicationTable2.cs
using System;

public class _11_MultiplicationTable2
{
    public static void Main(string[] args)
    {
        // Read input value
        int n1 = int.Parse(Console.ReadLine());
        int n2 = int.Parse(Console.ReadLine());

        // Print the multiplication based on the input
        if (n2 > 10)
        {
            Console.WriteLine($"{n1} X {n2} = {n1 * n2}");
        }
        else
        {
            for (int i = n2; i <= 10; i++)
            {
                Console.WriteLine($"{n1} X {i} = {n1 * i}");
            }
        }
    }
}

File: 02_ProgrammingFundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_01_Lecture/_11_MultiplicationTable2.java
package _02_ProgrammingFundamentals._01_BasicSyntaxConditionalStatementsAndLoops._01_Lecture;

import java.util.Scanner;

public class _11_MultiplicationTable2 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        int n1 = Integer.parseInt(scan.nextLine());
        int n2 = Integer.parseInt(scan.nextLine());

        // Print the multiplication based on the input
        if (n2 > 10) {
            System.out.printf("%d X %d = %d%n", n1, n2, n1 * n2);
        } else {
            for (int i = n2; i <= 10; i++) {
                System.out.printf("%d X %d = %d%n", n1, i, n1 * i);
            }
        }

        scan.close();
    }
}
